solution/Dockerfile.template

```diff
@@ -12,6 +12,9 @@ RUN pip install --no-cache-dir --upgrade pip setuptools \
 
 ENV SEARCHGOV_KEY=""
 ENV SEARCHGOV_SITE_NAME=""
+ENV BOX_CLIENT_ID=''
+ENV BOX_CLIENT_SECRET=''
+ENV BOX_REDIRECT_URL=''
 
 CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]
 
```

Comment:
If we did use box, we will need to update our Dockerfile to have these values. They are located in https://app.box.com/developers/console/app/2059452/configuration . 
If we use JWT token we would also need to add the location of the RSA file, JWT key and passphrase discussed here[ ](https://github.com/box/box-python-sdk#server-to-server-auth-with-jwt)